packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts

```diff
@@ -181,11 +181,15 @@ export class HostedPluginProcess implements ServerPluginRunner {
             return;
         }
         this.logger.error(`[${serverName}: ${pid}] IPC exited, with signal: ${signal}, and exit code: ${code}`);
-        this.messageService.error(
-            'Plugin runtime crashed unexpectedly, all plugins are not working, please reload the page. ' +
-            'It might happen if there is not enough memory for the plugins.',
-            { timeout: 15 * 60 * 1000 }
-        );
+
+        const message = 'Plugin runtime crashed unexpectedly, all plugins are not working, please reload the page.';
+        let hintMessage: string = 'If it doesn\'t help, please check Theia server logs.';
+        if (signal.toUpperCase() === 'SIGKILL' || signal.toUpperCase() === 'KILL') {
```

Comment:
I'm getting:

```
root ERROR TypeError: Cannot read property 'toUpperCase' of null
    at HostedPluginProcess.onChildProcessExit (/workspace/theia/packages/plugin-ext/lib/hosted/node/hosted-plugin-process.js:213:20)
    at ChildProcess. (/workspace/theia/packages/plugin-ext/lib/hosted/node/hosted-plugin-process.js:202:74)
    at Object.onceWrapper (events.js:277:13)
    at ChildProcess.emit (events.js:189:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
```

I test by:
- install VS Code extension
- launch `Launch Browser Backend`
- try to open 2 windows
- the second should have a notification

Reply:
Sorry, silly mistake...

Reply:
Fixed and tested